api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantServiceV3IntegrationTests.java

```diff
@@ -77,6 +89,78 @@ void shouldAcceptAndCreateANewValidClaimWithNoNullFields() throws JsonProcessing
         verifyPostToEligibilityService();
     }
 
+    @Test
+    void shouldReturnDuplicateStatusWhenEligibleClaimAlreadyExistsForDwpHouseholdIdentifier() throws JsonProcessingException {
+        //Given
+        String householdIdentifier = "dwpHousehold1";
```

Comment:
You can use the `DWP_HOUSEHOLD_IDENTIFIER` test constant, it is exactly this value.

Reply:
Done.